sdk/cosmos/azure-cosmos-spark_3_2-12/Samples/DatabricksLiveContainerMigration/README.md

```diff
@@ -0,0 +1,76 @@
+# Live Migrate Azure Cosmos DB SQL API Containers data with Spark Connector and Azure Databricks
+
+This sample shows how to live migrate between Azure Cosmos DB SQL API containers using the [Azure Cosmos DB Spark Connector](https://docs.microsoft.com/azure/cosmos-db/sql/create-sql-api-spark). 
+
+# Setup Azure Databricks
+
+## Prerequisites
+
+* [Provision an Azure Databricks cluster](https://docs.microsoft.com/azure/databricks/scenarios/quickstart-create-databricks-workspace-portal?tabs=azure-portal).
+
+* Ensure it has network access to your source and target Azure Cosmos DB Containers.
+
+
+## Provision a Spark cluster
+
+Select the latest Azure Databricks runtime version which supports Spark 3.0 or higher.
+
+![image](./media/spark-cluster.jpg)
+
+## Install Azure Cosmos DB Spark Connector jar
+
+* Install the Azure Cosmos DB Spark Connector jar on the cluster by providing maven co-ordinates `com.azure.cosmos.spark:azure-cosmos-spark_3-2_2-12:4.7.0`:
```

Comment:
Please use maven coordinates `com.azure.cosmos.spark:azure-cosmos-spark_3-2_2-12:latest` instead - that will always install the latest version - which avoids that we have to update it all the time

Reply:
Doesn't that also run the risk of latest jar containing some code that is not backwards compatible with the sample code at the time user installs it? Either way, I don't expect this sample to be very long lived (that said, I really don't know).

Reply:
Breaking changes should result in new major version - at which point we need to update all samples anyway - and any minor-verson upgrades are not supposed to break any workload just using public surface area - identifying it in a sample would be much better than when customers use it (but we have coverage for this migration scenario even in the CI pipeline)

Reply:
"identifying it in a sample would be much better than when customers use it" - ok, logical enough for me :-)

Reply:
Databricks failed to install the jar with maven coordinates version given as "latest". Can debug later, for now leaving at this version.